README.md

```diff
@@ -48,6 +50,11 @@ issues](https://github.com/kata-containers/kata-containers/issues/new):
 
 ## Kata Containers repositories
 
+### CI
+
+The [CI](https://github.com/kata-containers/ci) repository stores Continuous
+Integration (CI) configuration.
```

Comment:
maybe expand 'configuration' to 'configurations' or 'system configuration information' or similar

Reply:
Done.